cmd/ocm/cluster/status/status.go

```diff
@@ -58,19 +58,7 @@ func run(cmd *cobra.Command, argv []string) error {
 
 	cluster := response.Body()
 
-	//Get data out of the response
-	clusterMemory := cluster.Metrics().Memory()
-	clusterCPU := cluster.Metrics().CPU()
-	memUsed := clusterMemory.Used().Value() / 1000000000
-	memTotal := clusterMemory.Total().Value() / 1000000000
-
-	fmt.Printf("State:   %s\n"+
-		"Memory:  %.2f/%.2f used\n"+
-		"CPU:     %.2f/%.2f used\n",
-		cluster.State(),
-		memUsed, memTotal,
-		clusterCPU.Used().Value(), clusterCPU.Total().Value(),
-	)
```

Comment:
I think it is not OK to just remove these metrics. Can't this be replaced with the metrics obtained from the accounts management service?

Reply:
Added